### ISSUE TITLE
CI/CD test AlaSQL in older node versions with no build step

### ISSUE THREAD

**Issue by mathiasrw** [AlaSQL/alasql#1583]
Let the CICD run the tests of AlaSQL on older versions of Node - without the build step (as node v14 is needed to build)

**Comment by jimmywarting** [comment]
What node version do you want to run on?

**Comment by mathiasrw** [comment]
For building? Does not matter. I propose the latest LTS version. Currently v18 as I recall.

**Comment by mathiasrw** [comment]
Im thinking: Use nvm in the github action to build with v18 and then change the node version and run for each version we want to run on. Its not because we cant break with old versions. Its more to be able to tell people "not this is no longer used". We still have heaps of node 0.12 running on physical boards (probably using older versions anyway) but its nice to say "hey, if you want to use this node version, you have to stick to this version of Alasql)

I propose we run the test for 

- v0.10
- v0.12
- v10
- v11
- v12
- v13

**Comment by mathiasrw** [comment]
Actually. It got really messy with all the tests needing to be transpiled - and I realised we would have to transpile alasql.fs.js also. We might do that at some piut, but for now I must admit that we only support node v15+

It did bring me to some interesting understanding of the github actions. 

I have prepared the structure for when we are ready with a package.js command that can run the browser tests so we have both node and browser tests running on the same build. 

https://github.com/AlaSQL/alasql/commit/5e394233cf79ad55b91f29f9afff0610d1e03539#diff-71ddae2314d21673b588320a34327a6901b68a61ad7528957317cf3b84fd3accR63
